File: Modul-5/Soal-4/PRAK504-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>
int reverse(int bil){
    int rev = 0;
    while (bil > 0) {
        int sisa = bil % 10;
        rev = rev * 10 + sisa;
        bil /= 10; 
    }
    return rev;
}
int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));
}

File: Modul-5/Soal-4/PRAK504-2410817110016-MuhammadAzriRahman.py
def reverse(bil):
    rev = 0
    while (bil > 0):
        sisa = bil % 10
        rev = rev * 10 + sisa
        bil //= 10
    return rev

if __name__ == "__main__":
    A, B = map(int, input("").split())

    A = reverse(A)
    B = reverse(B)
    C = A+B

    print(reverse(C))
